src/olympia/accounts/views.py

```diff
@@ -774,3 +778,88 @@ def post(self, request):
                 _('Notification [%s] does not exist') % notification_name
             )
         return Response(serializer.data)
+
+
+class FxaNotificationView(FxAConfigMixin, APIView):
+    authentication_classes = []  # TODO: reimplement FxA auth in authentication_class?
+    permission_classes = []
+
+    FXA_PROFILE_CHANGE_EVENT = (
+        'https://schemas.accounts.firefox.com/event/profile-change'
+    )
+    FXA_DELETE_EVENT = 'https://schemas.accounts.firefox.com/event/delete-user'
+    FXA_PASSWORDCHANGE_EVENT = (
+        'https://schemas.accounts.firefox.com/event/password-change'
+    )
+
+    def get_fxa_verifying_keys(self):
+        if not hasattr(self, 'fxa_verifying_keys'):
+            response = requests.get(f'{settings.FXA_OAUTH_HOST}/jwks')
+            self.fxa_verifying_keys = (
+                response.json().get('keys') if response.status_code == 200 else []
+            )
+
+        if not self.fxa_verifying_keys:
+            raise exceptions.AuthenticationFailed(
+                'FXA verifying keys are not available.'
+            )
+
+        return self.fxa_verifying_keys
+
+    def get_jwt(self, request):
+        request_jwt = request.headers['Authorization'].split('Bearer ')[1]
```

Comment:
Probably want a 40x when there is no `Authorization` header, or when the split value length is not 2.